samples/subsys/mgmt/mcumgr/smp_svr/README.rst

```diff
@@ -149,16 +149,16 @@ To sign the sample image we built in a previous step:
 
     west sign -t imgtool -- --key bootloader/mcuboot/root-rsa-2048.pem
 
-The above command creates an image file called :file:`zephyr.signed.bin` in the
-build directory.
+The above command creates signed image files called :file:`zephyr.signed.bin` and
+:file:`zephyr.signed.hex` in the build directory.
 
 For more information on image signing and ``west sign``, see the :ref:`west-sign`
 documentation.
 
 Flashing the sample image
 *************************
 
-Upload the :file:`zephyr.signed.bin` file from the previous to image slot-0 of your
+Upload the :file:`zephyr.signed.hex` file from the previous to image slot-0 of your
```

Comment:
It might be either *.bin or *.hex file. For instance west flash uses nrfjprog script and *.hex file for nRF devices.